### DISCUSSION TITLE
tailwind css not working after all configrations in vue js

### DISCUSSION THREAD

**Discussion by surajkr214** [tailwindlabs/tailwindcss#11962] [Help]
i have already set up the vue js and installed tailwind css and the tailwind css intelligence too.
although tailwind css is not working

**Comment by wongjn** [comment]
Consider checking the [framework installation guide](https://tailwindcss.com/docs/installation/framework-guides) most suitable for your project to check you have followed all the necessary steps correctly. If you have checked and things still aren't working, consider providing a remotely-hosted git repository that reproduces the unexpected behavior.
